test/jdk/javax/sound/midi/Sequencer/MetaCallback.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2017, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
futuristic year? or you didnt have confidence that this test will be integrated this year :-)

Reply:
> futuristic year? or you didnt have confidence that this test will be integrated this year :-)

Is this still 2022? I guess i just wish this year is over already. Will fix that though.